src/main/java/com/stripe/model/Event.java

```diff
@@ -31,6 +31,14 @@
   @Deprecated
   String userId;
 
+  /**
+   * Get event data with explicit deserialization support. This object contains superset
+   * information of {@link EventData}.
+   */
+  public EventVersionedData getVersionedData() {
```

Comment:
Entry point to the new proxy object that is preferred over the original `EventData`